docs/source/table.rst

```diff
@@ -129,6 +129,17 @@
      - 252
      - :ikala:`\ `
 
+   * - Haydn op20
+     - - audio: ❌
+       - midi: ✅
+       - scores: ✅
+       - annotations: ✅
+     - - symbolic :ref:`chords`
```

Comment:
I don't think you need to label them as `symbolic`, just `chords` and `key` should be fine!

Reply:
@nkundiushuti and @magdalenafuentes recommended it. Because it doesnt have a "usual time". It has "score relative time": the music21 offsets